Lib/http/server.py

```diff
@@ -1362,7 +1362,7 @@ def test(HandlerClass=BaseHTTPRequestHandler,
     parser.add_argument('port', default=8000, type=int, nargs='?',
                         help='bind to this port '
                              '(default: %(default)s)')
-    args = parser.parse_args()
+    args = parser.parse_args(args=args)
```

Comment:
it's not big diff but let's stick to single style
```suggestion
    args = parser.parse_args(args)
```